changelogs/fragments/705-pg_hba-invalid-and-parsing.yml

```diff
@@ -0,0 +1,3 @@
+bugfixes:
+  - postgresql_pg_hba - fixes #705 by preventing invalid strings to be written
```

Comment:
this needs to be quoted to not become a yaml comment

Reply:
fixed in 08c5008